testgrid/specs/full.yaml

```diff
@@ -132,18 +132,16 @@
   - centos-83
   - centos-84
   - ol-84
-- name: k8s118
+- name: k8s120x_docker
   installerSpec:
     kubernetes:
-      version: 1.18.x
+      version: 1.20.x
     docker:
       version: 19.03.10
     weave:
       version: 2.6.5
     contour:
       version: 1.0.1
-    rook:
-      version: 1.0.4
```

Comment:
Keep this - `kotsadm` will need a CSI provider to allocate storage.

Reply:
Can `kotsadm` use minio as storage provider?

Reply:
Switching to `minio`